s3_sync/s3_sync.py

```diff
@@ -33,126 +34,156 @@
 config.read(settings_file)
 
 # Configure logbook logging
-logger = RotatingFileHandler(config['Logs']['logfile'], max_size = int(config['Logs']['max_size']),
-	backup_count = int(config['Logs']['backup_count']),
-	level = int(config['Logs']['level']))
+logger = RotatingFileHandler(config['Logs']['logfile'],
+                             max_size = int(config['Logs']['max_size']),
+                             backup_count = int(config['Logs']['backup_count']),
+                             level = int(config['Logs']['level']))
 logger.push_application()
 logger = Logger(__name__)
 
 # Get Computer name
 computer_name = os.environ['COMPUTERNAME']
 
+
 def verify_settings_file_exists(settings_file):
-	if os.path.exists(settings_file):
-		return
-	else:
-		logger.error("Settings file not found:", settings_file)
-		sys.exit("[-] Settings file not found")
+    """
+    Verify that settings file exists.
+
+    Args:
+      settings_file (str): Path for the local settings file.
+
+    Returns:
+      If file exists, return None, otherwise exit.
+    """
+    if os.path.exists(settings_file):
+        return
+    else:
+        logger.error("Settings file not found:", settings_file)
+        sys.exit("[-] Settings file not found")
+
 
 def set_environment_variables():
-	os.environ['AWS_ACCESS_KEY_ID'] = config['AWS']['AWS_ACCESS_KEY_ID']
-	os.environ['AWS_SECRET_ACCESS_KEY'] = config['AWS']['AWS_SECRET_ACCESS_KEY']
-	os.environ['slack_webhook_url'] = config['Slack']['webhook_url']
+    """
+    Set some of the read settings as environment variables.
+    """
+    os.environ['AWS_ACCESS_KEY_ID'] = config['AWS']['AWS_ACCESS_KEY_ID']
+    os.environ['AWS_SECRET_ACCESS_KEY'] = config['AWS']['AWS_SECRET_ACCESS_KEY']
+    os.environ['slack_webhook_url'] = config['Slack']['webhook_url']
+
 
 def verify_local_folder_exists(local_video_records_done_folder):
-	"""
+    """
     Check whether the local video records done folder exists
 
     Args:
       local_video_records_done_folder (str): The path of the local video
-      	records done folder.
+        records done folder.
 
     Returns:
       If folder exists return None, and if not, logs error and exit.
-	"""
-	if os.path.exists(local_video_records_done_folder):
-		return
-	else:
-		logger.exception("Local Video Records Done folder not found")
-		sys.exit("[-] Local Video Records Done folder not found")
+    """
+    if os.path.exists(local_video_records_done_folder):
+        return
+    else:
+        logger.exception("Local Video Records Done folder not found")
```

Comment:
I thought if you do logger.error it would not include a stack trace of the error in the log file or is that not accurate?